### PR TITLE
Add some text to README

### PR DESCRIPTION
**Pull request by peterjc**
My intent is to help anyone looking to file an issue.

Checklist
* [X] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+This repository https://github.com/conda-forge/feedstocks/ does not get updated by hand,
+rather it is an automated summary of all the individual package feedstocks which have
+their own separate repositories.
+
+If you want to file an issue with an individual package, say ``unzip``, go to the
+https://github.com/conda-forge/unzip-feedstock/issues page.
+
+If you want to file a wider issue, like requesting a new package be added, go to the
+https://github.com/conda-forge/staged-recipes/issues page.
```
Comment (scopatz) [inline README.md]: Wider issues should be filed at https://github.com/conda-forge/conda-forge.github.io, not staged-recipes, though you are right about package requests

Comment (peterjc) [reply README.md]: I can change that - or is there a good general URL we should give instead?

Comment (scopatz) [reply README.md]: Well, it is the issues page on that repo if you want to use that

**Comment by jakirkham** [comment]
Yeah agree with @scopatz. Otherwise looks good. Thanks! 😀

**Comment by scopatz** [comment]
Thanks @peterjc!

**Comment by peterjc** [comment]
Thanks both - this should help a few lost people 👍

**Pull request merged**